src/yuzu/main.cpp

```diff
@@ -198,6 +198,36 @@ static void RemoveCachedContents() {
     Common::FS::RemoveDirRecursively(offline_system_data);
 }
 
+static void LogRuntimes() {
+#ifdef _MSC_VER
+    // It is possible that the name of the dll will change.
+    // vcruntime140.dll is for 2015 and onwards
+    constexpr char runtime_dll_name[] = "vcruntime140.dll";
+    // winver.h calls this dwHandle, but the parameter isn't used as a handle of any kind.
+    DWORD dw_handle;
+    uint sz = GetFileVersionInfoSizeA(runtime_dll_name, &dw_handle);
```

Comment:
```suggestion
    UINT sz = GetFileVersionInfoSizeA(runtime_dll_name, &dw_handle);
```

is the proper Windows type alias